internal/backend/circuits/circuits.go

```diff
@@ -39,5 +65,5 @@ func addNewEntry(name string, circuit frontend.Circuit, proverGood, proverBad []
 	}
 	solver.RegisterHint(hintFunctions...)
 
-	Circuits[name] = TestCircuit{circuit, proverGood, proverBad, hintFunctions, curves}
+	Circuits[name] = TestCircuit{Circuit: circuit, ValidAssignments: proverGood, InvalidAssignments: proverBad, HintFunctions: nil, Curves: curves}
```

Comment:
I think I forgot to pass `hintFunctions` here

Reply:
And due to that, I think we can also remove the prover/verifier options for the integration tests.